test/EnliteMonologTest/Service/FormatterMock.php

```diff
@@ -1,46 +1,92 @@
 <?php
 
+// @codingStandardsIgnoreFile
```

Comment:
what are these about?

Reply:
To keep compatibility I needed to create conditional implementations of mock. After that PHPCS throw this error https://travis-ci.org/enlitepro/enlite-monolog/jobs/641132116#L321

Reply:
i see. i suppose it will be short-lived anyway. we can drop `monolog:^1` support in the next major version of this package.